worlds/Arena.wbt

```diff
@@ -2,8 +2,9 @@
 WorldInfo {
 }
 Viewpoint {
-  orientation 0.33246481952953133 -0.9314016676496985 -0.14818258087426103 5.456433700612629
-  position 6.9972994812935 3.16953523821157 7.1575439818618
+  orientation -0.20519757298593996 -0.9759022170490246 -0.07422141739065735 0.8170417753010643
+  position -8.955236326402577 3.7554386590715487 8.682878887529787
+  follow "S35"
```

Comment:
Is this change relevant or can we drop it?

Reply:
Will be dropped -  left over view point changes when replacing tokens